src/webrtc/call.js

```diff
@@ -593,6 +584,27 @@ MatrixCall.prototype._maybeGotUserMediaForInvite = function(stream) {
     setState(self, 'create_offer');
 };
 
+MatrixCall.prototype._sendAnswer = function(stream) {
+    sendEvent(this, 'm.call.answer', this._answerContent).then(() => {
+        setState(this, 'connecting');
+        // If this isn't the first time we've tried to send the answer,
+        // we may have candidates queued up, so send them now.
+        _sendCandidateQueue(this);
+    }).catch((error) => {
+        // We've failed to answer: back to the ringing state
+        setState(this, 'ringing');
+        this.client.cancelPendingEvent(error.event);
+        this.emit(
```

Comment:
surely we should only emit this if the error is UnkDevices?

Reply:
(it's a bit unintuitive also that the fact there are unknown devices in the room gets in the way of call signalling, which the user might reasonably expect to be 1:1. it's almost an argument that we should be using toDevice messaging for the 1:1 bits of VoIP Signalling...)

Reply:
oops, thanks!

Reply:
and yeah, although in the case of an invite you'll still want to send it to all devices. For answers it is a bit silly since it only needs to go to the device the invite came from.